enhancements/console/dynamic-plugins.md

```diff
@@ -185,63 +204,122 @@ extension points will be available.
 
 ```json
 {
-  "$schema": "/path/to/schema/manifest.json",
-  "id": "my-plugin",
+  "name": "my-plugin",
   "version": "1.2.3",
-  "pluginAPI": "4.7.x",
   "displayName": "My Plugin",
+  "dependencies": {
+    "@console/pluginAPI": "*"
```

Comment:
Hm, this should probably be `4.7.x` instead of `*` since the latter is not very explanatory.